packages/dashboards/app/styles/navi-dashboards/components/dashboard-header.scss

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2021, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+
+.dashboard-header {
+
+  &__page-title {
+    font-size: 30px;
+    text-transform: capitalize;
+    display: flex;
+    min-width: 0;
+  }
+
+  &__fav-icon {
+    margin: 3px 10px;
+    .fa {
+      font-size: 18px;
+    }
+  }
+
+  &__actions {
+    display: flex;
+    @media (min-width: 0px) and (max-width: 899px) {
+      justify-content: flex-start;
+      .button {
+       justify-content: left;
+      }
+    }
+    @media (min-width: 900px) {
```

Comment:
Can we use the Denali mixins: https://denali.design/docs/2.2.3/layouts/breakpoints